README.md

```diff
@@ -597,6 +597,20 @@ public void CanEnter(Age age) {
 }
 ```
 
+### Can I create custom value object attributes with my own defaults?
+
+Yes, but (at the moment) it requires that you put your defaults in your attribute's primary constructor - not in the call to the base class' constructor (see [this comment](https://github.com/SteveDunn/Vogen/pull/321#issuecomment-1399324832)).
```

Comment:
Yeah, interesting! I assume (because I haven't checked the lowered code) that base constructor calls get moved into the constructor's body. 

I'm interested in getting some compile-time execution of these attributes using something hacky (maybe the `CSharpScript` class?) so these can be extracted from fields instead. Definitely not something Roslyn can do off the bat.